wpilib/wpilib/timedrobot.py

```diff
@@ -25,17 +25,15 @@ class TimedRobot(IterativeRobotBase):
 
     periodic() functions from the base class are called on an interval by a Notifier instance.
     """
-    DEFAULT_PERIOD = .02
+    kDefaultPeriod = .02
 
 
-    def __init__(self):
-        super().__init__()
-        hal.report(hal.UsageReporting.kResourceType_Framework, hal.UsageReporting.kFramework_Iterative)
+    def __init__(self, period: float=None):
+        if period is None:
+            period = TimedRobot.kDefaultPeriod
```

Comment:
I think the default should be in the method signature here. Would you agree @virtuald?

Reply:
I agree.